### PR TITLE
utils/add_platform_rule.py: Fix create option and enhance it

### PR DESCRIPTION
**Pull request by JAORMX**
This fixes the `create` option that had gotten out of date with the
latest kubectl enhancements (specially rate-limitting comments on the
logs), and it also adds a couple of features:

* Makes the name optional (enabling us to list objects)
* Adds an `--all-namespaces` option

Signed-off-by: Juan Antonio Osorio Robles <jaosorior@redhat.com>

### CODE REVIEW

File: utils/add_platform_rule.py [hunk 1]
```diff
@@ -359,7 +369,7 @@ def main():
     create_parser.add_argument(
         '--group', default="", help='The group directory of the rule to create.')
     create_parser.add_argument(
-        '--name', required=True, help='The name of the Kubernetes object to check. Required.')
+        '--name', help='The name of the Kubernetes object to check. Required.')
```
Comment (Vincent056) [inline utils/add_platform_rule.py:372]: Do we also remove `Required` from `help`

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now fb15af4010ff278b42a551370089e6cb528a6396

File: utils/add_platform_rule.py (hunk 1 above)
Comment (JAORMX) [reply utils/add_platform_rule.py]: Thanks! Done

**Review by mrogers950 — APPROVED** [review]
/lgtm

**Pull request merged**